taiga_halo2/src/circuit/vp_circuit.rs

```diff
@@ -56,6 +55,8 @@ use vamp_ir::halo2::synth::{make_constant, Halo2Module, PrimeFieldOps};
 use vamp_ir::transform::compile;
 use vamp_ir::util::{read_inputs_from_file, Config};
 
+pub type ValidityPredicate = dyn ValidityPredicateVerifyingInfo;
+
```

Comment:
Yeah this is a fine simplification, seems it is still used everywhere, however this does bring up a question.